packages/components/src/Glimmer/Glimmer.css

```diff
@@ -5,6 +5,11 @@
   --duration-base: calc(var(--timing-slowest) * 4); /* 2s */
   --duration-fast: calc(var(--duration-base) / 2); /* 1s */
 
+  @media screen and (prefers-color-scheme: dark) {
+    --glimmer-background: rgba(var(--color-white--rgb), 10%);
```

Comment:
Looking at this again I think I want to just do this with semantic colors instead of media queries

Reply:
Not a great way to do this, so I'm going to remove this change for now